crates/diesel_models/src/payment_intent.rs

```diff
@@ -365,6 +365,11 @@ pub enum PaymentIntentUpdate {
         status: storage_enums::IntentStatus,
         updated_by: String,
     },
+    /// Update the payment intent details on payment intent session token call, before calling the connector
+    MetadataUpdate {
+        metadata: masking::Secret<serde_json::Value>,
```

Comment:
This is the metadata that is sent by merchant in intent create call. What are we updating it in sessions call?